src/pages/workspaces/List.js

```diff
@@ -47,40 +47,6 @@ const workspaceSubmissionStatus = ({ workspaceSubmissionStats: { runningSubmissi
   )
 }
 
-const WorkspaceMenuContent = ({ namespace, name, onClone, onShare, onDelete, onLock }) => {
-  const { workspace } = useWorkspaceDetails({ namespace, name }, ['accessLevel', 'canShare', 'workspace.isLocked'])
-  const canRead = workspace && Utils.canRead(workspace.accessLevel)
-  const canShare = workspace?.canShare
-  const isOwner = workspace && Utils.isOwner(workspace.accessLevel)
-  const isLocked = workspace?.workspace.isLocked
-  return h(Fragment, [
-    h(MenuButton, {
-      disabled: !canRead,
```

Comment:
We actually never created this menu if the user does not have read permissions for the workspace, so there was no need to carry the check for `canRead` into the shared code.

![image](https://user-images.githubusercontent.com/484484/167696177-6ce7f4af-57e5-4827-9e80-330eb987e2d3.png)